share/scripts/sdcdb.tcl

```diff
@@ -360,14 +360,16 @@ proc read_cdb {fname} {
     close $fh
     variable c_files
     variable a_files
-    puts "[array size c_files] C files added"
-    puts "[array size a_files] assembly files added"
-    puts "[array size c_files_count] C files references added"
-    puts "[array size a_files_count] assembly files references added"
-    puts "$c_count C source lines found"
-    puts "$a_count assembly source lines found"
-    puts "$gf_count global function(s) registered"
-    puts "$sf_count static function(s) registered"
+    return [subst [join {
+        "[array size c_files] C files added"
+        "[array size a_files] assembly files added"
+        "[array size c_files_count] C files references added"
+        "[array size a_files_count] assembly files references added"
+        "$c_count C source lines found"
+        "$a_count assembly source lines found"
+        "$gf_count global function(s) registered"
+        "$sf_count static function(s) registered"
+    } "\n" ]]
```

Comment:
This is perfectly fine. No need to change (I mean pick what you like better).
My idea was to write it like:
```
set result ""
append result "....\n"
append result "....\n"
return $result
```